### PR TITLE
LinearAlgebra: Add test "matrix x matrix with negative stride".

### PR DESCRIPTION
**Pull request by ghost**
(Made while looking at #39836, which is matrix x vector.
Matrix x matrix with negative stride is fine as it is, because
it uses the Julia implementation of matrix multiplication.)

### CODE REVIEW

File: test/cmdlineargs.jl [hunk 1]
```diff
@@ -206,6 +206,7 @@ let exename = `$(Base.julia_cmd()) --startup-file=no --color=no`
     # We want to test oversubscription, but on manycore machines, this can
     # actually exhaust limited PID spaces
     cpu_threads = max(2*cpu_threads, min(50, 10*cpu_threads))
+    println("cpu_threads = ", cpu_threads, ", Sys.WORD_SIZE = ", Sys.WORD_SIZE)
```
Comment (simeonschaub) [inline test/cmdlineargs.jl:209]: This is a known failure unrelated to this PR, so not neccessary to debug this here.

Comment (dagbr) [reply test/cmdlineargs.jl:209]: @simeonschaub Thanks, could you point me to the place this is discussed?

**Comment by dagbr** [comment]
(back to just one commit which adds a test)

File: test/cmdlineargs.jl (hunk 1 above)
Comment (simeonschaub) [reply test/cmdlineargs.jl]: Should now be fixed by https://github.com/JuliaLang/julia/pull/39854

**Pull request merged**